src/com/createsend/models/journeys/JourneyEmailSummary.java

```diff
@@ -0,0 +1,12 @@
+package com.createsend.models.journeys;
+
+public class JourneyEmailSummary {
+    public String EmailID;
+    public String Name;
+    public int Sent;
+    public int Opened;
+    public int Clicks;

```

Comment:
Did we decide not to change this to Clicked?

Reply:
the actual endpoint has not been changed to `Clicked` yet - we'll update this when the endpoint is changed